app/creator/[address]/page.tsx

```
@@ -0,0 +1,168 @@
+      {/* Videos Section */}
+      <div>
+        <h2 className="mb-4 text-2xl font-bold">
+          {videoCount > 0 ? `All Videos (${videoCount})` : "All Videos"}
+        </h2>
+        {videoCount === 0 ? (
+          <Card>
+            <CardContent className="flex flex-col items-center justify-center py-12">
+              <Video className="h-12 w-12 text-muted-foreground mb-4" />
+              <p className="text-lg font-semibold mb-2">No videos yet</p>
+              <p className="text-sm text-muted-foreground text-center">
+                This creator hasn't uploaded any videos yet.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)